### ISSUE TITLE
Dimensions of REPR_DIM_DICT and smplx_params_dict

### ISSUE THREAD

**Issue by Xianqi-Zhang** [sanweiliti/RoHM#14]
Hello,
I want to ask a question about dimensions.

#### 1. REPR_DIM_DICT
As presented in section 3,  subsection `Motion Representation`,  local joint positions J ∈ R21×3,  joint rotations θ ∈ R21×6, but in REPR_DIM_DICT, `local_postions` and `local_vel` are defined as 22×3 (for 22 body joints ?).
Does this is because the pelvis joint is used as the origin of the coordinate system, so 22 -> 21 ?

#### 2. smplx_params_dict
As illustrated in paper [Expressive Body Capture: 3D Hands, Face, and Body from a Single Image](https://openaccess.thecvf.com/content_CVPR_2019/papers/Pavlakos_Expressive_Body_Capture_3D_Hands_Face_and_Body_From_a_CVPR_2019_paper.pdf), section3.1, `The total number of model parameters iS MPL-X is 119: 75 for the global body rotation and { body, eyes , jaw } joints, 24 parameters for the lower dimensional hand pose PCA space, 10 for subject shape and 10 for the facial expressions.` 

But I can't match these values ​​with the values ​​in the paper, e.g., `24 parameters for the lower dimensional hand pose PCA space`, but in this repo, ` smplx_params_dict_noisy_torch['right_hand_pose'] = torch.zeros(bs, 45).to(self.device)`.

I only found a simple description of the dimensions in [issue](https://github.com/vchoutas/smplify-x/issues/170), i.e., `SMPLX original param dimension: betas 10, global_orient 3, transl 3, left_hand_pose 45, right_hand_pose 45, jaw_pose 3, leye_pose 3 , reye_pose 3, expression 10, body_pose 63, keypoints_3d 25, pose_embedding 32, gender 1, v 10475`, but I can't understand the specific meaning of these dimensions, such as why left_hand_pose is 45 (maybe 5 fingers * 9 rotation matrix ?), and what's the meaning of transl (camera translation ?)...

Could you explain the meaning and dimensions of these variables ? 
Or point me some tutorial ?

Thanks for your any reply.

Best regards.

------------------------------------------------------------------

https://github.com/sanweiliti/RoHM/blob/6afeb566bc66626cd3b5f7fdcba14b8fe13f7c6f/utils/other_utils.py#L24-L37
https://github.com/sanweiliti/RoHM/blob/6afeb566bc66626cd3b5f7fdcba14b8fe13f7c6f/data_loaders/dataloader_amass.py#L199-L206
https://github.com/sanweiliti/RoHM/blob/6afeb566bc66626cd3b5f7fdcba14b8fe13f7c6f/data_loaders/motion_representation.py#L379-L388

**Comment by Xianqi-Zhang** [comment]
1. It seems that, RoHM uses a total of 21 body joints with 1 joint for global rotation, not "a total of 22 body joints".
(as illustrated in SMPL-X paper, subsection 3.1, "..., parameterized by the pose $θ^2 \in R^{3(K+1)}$, K is the number of body joints in addition to a joint for global rotation." )

2. SMPL-X paper, subsection 3.1, "The fingers have 30 joints, which correspond to 90 pose parameters (3 DoF per joint as axis-angle rotations).", so the dimension of "left_hand_pose" and "right_hand_pose" are 45.

**Comment by Xianqi-Zhang** [comment]
If you have any questions about SMPL-X, you can refer to the following PPT. 
I hope it will be helpful to you...
https://files.is.tue.mpg.de/black/talks/SMPL-made-simple-FAQs.pdf
